### DISCUSSION TITLE
v1.6.7

### DISCUSSION THREAD

**Discussion by deniscerri** [deniscerri/ytdlnis#282] [General]
## What's New

- Beta Releases will be on Github from now on, unless there is a small thing you will find quick builds on telegram
- Fixed Terminal not showing shared link
- Fixed Terminal sometimes not showing the progress text
- Changed Terminal share logo color
- Aded javanese language
- Added bengali india language
- Fixed app adding bestaudio in format even if remove audio is set
- Made app fetch only link parts from sources like pinterest that also include text in front
- Added ability to switch to the beta channel for updating. You can also downgrade to the latest release version
- Fixed app not restarting when changing language
- Added word wrap toggles for logs and terminal
- Added Auto Preferred Download Type. It will open as Audio if the site is known to post audio files and video for the rest
- Added collected file size in multi download card
- Fixed app not selecting best format
- Fixed Youtube Music Playlist not loading
- Fixed Sponsorblock not showing as checked in multiple download card #271
- Added playlist title and playlist index metadata in playlist items so u can use them in filename templates #272 #270
- Added -a path.txt if you share a text file to the app and the preferred download type is command or terminal
- Fixed app cancelling workmanager earlier than expected
- Added title and author sync between audio and video tabs #241
- Fix format details fps in arabic being rtl
- Removed h264 from codec
- Fixed Search engine not getting updated in home when changing in settings fragment.
- Added language preference in android 13 and up
- Added multiple selection for command templates
- Fix issue when app crashes when creating config files for items with weird titles #276 
- Add multiline titles of preferences
- Fix app conflicting format id with a generic format id
- Finally shrunk the size down lol

## Autogenerated Changes
* Translations update from Hosted Weblate by @weblate in https://github.com/deniscerri/ytdlnis/pull/266
* Translations update from Hosted Weblate by @weblate in https://github.com/deniscerri/ytdlnis/pull/269
* Better translation by @azf3 in https://github.com/deniscerri/ytdlnis/pull/280
* Translations update from Hosted Weblate by @weblate in https://github.com/deniscerri/ytdlnis/pull/279
* Translations update from Hosted Weblate by @weblate in https://github.com/deniscerri/ytdlnis/pull/281

## New Contributors
* @azf3 made their first contribution in https://github.com/deniscerri/ytdlnis/pull/280

**Full Changelog**: https://github.com/deniscerri/ytdlnis/compare/v1.6.6...v1.6.7

<hr /><em>This discussion was created from the release <a href='https://github.com/deniscerri/ytdlnis/releases/tag/v1.6.7'>v1.6.7</a>.</em>

**Comment by apathymiller** [comment]
Which 1 do I need to download so that I can download yt vids? Just the most current 1, or do I need to start with another and then the most current? Sorry, I don't know a whole lot about computers/ software...

**Reply by deniscerri** [reply]
Try the arm64 one. If it doesn't work try the armv7 one

**Comment by apathymiller** [comment]
Tyvm, got it installed and working fine.  Have a great 1.

**Comment by Mullemaus-1992** [comment]
😋

**Comment by NighTMare-010** [comment]
Is there a way to download the synced lyrics of the music videos using this app.
Few months ago I used snaptube and it downloaded the synced lyrics of the music videos as well, is there any way to do the same using this app?

Thanks for this app btw🙂

**Reply by deniscerri** [reply]
This is still an open issue in ytdlp
https://github.com/yt-dlp/yt-dlp/issues/5635
